raijin_scripts/execute_sync/run

```diff
@@ -102,5 +102,5 @@ echo "Starting Sync process......"
 ##################################################################################################
 # Run dea-sync process
 ##################################################################################################
-qsub -V -N dea-sync -q "${QUEUE}" -W umask=33 -l wd,walltime=10:00:00,mem=25GB,ncpus=1 -P "${PROJECT}" -o "$WORKDIR" -e "$WORKDIR" -- dea-sync -vvv --cache-folder "$WORKDIR"/cache -j 4 --log-queries "$TRASH_ARCHIVED" --update-locations --index-missing "$PATH_TO_PROCESS"
+qsub -V -N dea-sync -q "${QUEUE}" -W umask=33 -l wd,walltime=20:00:00,mem=25GB,ncpus=1 -P "${PROJECT}" -o "$WORKDIR" -e "$WORKDIR" -- dea-sync -vvv --cache-folder "$WORKDIR"/cache -j 4 --log-queries "$TRASH_ARCHIVED" --update-locations --index-missing "$PATH_TO_PROCESS"
```

Comment:
Same here, I'd prefer to keep the limits a bit lower, at least until we work out how much is required, and have made sure it runs efficiently.

Half of the `v10` quota has been used already this quarter, not necessarily by us, but we drink from the same bucket.

Reply:
Revert back changes